### DISCUSSION TITLE
Archival format where local media is referenced?

### DISCUSSION THREAD

**Discussion by BlipRanger** [ArchiveBox/ArchiveBox#684] [Q&A] [answered]
Is there (or could there be) an archive format where the media (like a video/image) is referenced locally rather than on the web? I just noticed that when I archive a reddit post with a video on it, if I go offiline, the only way for me to view the video is via the media outside of the page itself.

**Comment by pirate** [comment, marked as answer]
The only tool I know of that can do this at the moment is ArchiveWeb.page / webrecorder.io. I already have plans to build it in as an extractor, but don't expect it anytime soon because it's a quite a bit of work.

**Reply by BlipRanger** [reply]
Sure, that makes sense, sounds good!
